flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/table/JdbcDynamicTableFactory.java

```diff
@@ -324,6 +332,13 @@ private void validateConfigOptions(ReadableConfig config) {
 				SINK_MAX_RETRIES.key(),
 				config.get(SINK_MAX_RETRIES)));
 		}
+
+		if (config.get(MAX_RETRY_TIMEOUT).getSeconds() <= 0) {
+			throw new IllegalArgumentException(String.format(
+				"The value of '%s' option must be in second granularity and shouldn't be smaller than 1 second, but is %s.",
+				MAX_RETRY_TIMEOUT.key(),
+				config.get(ConfigOptions.key(MAX_RETRY_TIMEOUT.key()).stringType().noDefaultValue())));
```

Comment:
I'm not sure the second placeholder maybe `config.get(MAX_RETRY_TIMEOUT).getSeconds()` instead of `config.get(ConfigOptions.key(MAX_RETRY_TIMEOUT.key()).stringType().noDefaultValue()))`.

Reply:
I had used  config.get(MAX_RETRY_TIMEOUT).getSeconds()  ,In fact config.get(ConfigOptions.key(MAX_RETRY_TIMEOUT.key()).stringType().noDefaultValue())) is right